python/lsst/dbb/buffmngrs/endpoint/finder/finder.py

```diff
@@ -304,11 +311,12 @@ def parse_rsync_logs(directory, blacklist=None,
     if blacklist is None:
         blacklist = []
     delay = timedelta(seconds=delay)
-    end = date.today()
+    origin = date.today()
     if isodate is not None:
-        end = isodate
-    dates = [end - timedelta(days=n) for n in range(timespan+1)]
-    for day in dates:
+        origin = isodate
+    past = [origin - timedelta(days=n) for n in range(1, past_days+1)]
+    future = [origin + timedelta(days=n) for n in range(1, future_days+1)]
+    for day in sorted([*past, origin, *future]):
```

Comment:
Didn't see anywhere where past and future are used separately.  Can't the day list be made in single loop and without sorting?  Or even just loop on day offset making the particular date inside the loop since the list of dates doesn't seem to be used elsewhere.